projects/project-player-library/src/lib/assets/i18n/en.json

```diff
@@ -84,6 +84,10 @@
     "INVALID_LINK_MSG":"Please check the link. Special characters, comma, spaces or new lines are not allowed.",
     "SUBMIT_IMPROVEMENT_CONFIRMATION_MSG":"Are you sure you want to submit your improvement project?. Once submitted no further edits can be made to the project.",
     "SUBMIT":"Submit",
+    "LOAD_MORE":"Load more",
+    "SUBTASK_DELETED":"Sub task deleted",
     "SYNCING_PAGE":"Syncing page",
-    "FILE_NOT_SHARED": "File not shared !"
+    "FILE_NOT_SHARED": "File not shared !",
+    "FILES_CHANGES_NOT_UPDATED":"File changes not updated !",
+    "LEARNING_RESOURCES_BACK":"Learning resources"
```

Comment:
Give relevant key names, give it as "LEARNING_RESOURCES"

Reply:
Done